manifests/manifests_test.go

```diff
@@ -0,0 +1,47 @@
+package manifests_test
+
+import (
+	"bytes"
+	"fmt"
+	"maistra.io/api/manifests"
+	"testing"
+)
+
+func TestGettingManifestDirEntries(t *testing.T) {
+	manifests, err := manifests.GetManifests()
+	if err != nil {
+		panic(fmt.Sprintf("Unexpected error: %v", err))
```

Comment:
t.Fatal() instead of panic() here and below?

Reply:
It's not a failure of the test though, it's an error of the test fixture.

Reply:
You are consuming the API and it returned an error, so...

Reply:
3f21fce